frontend/src/sass/components/_slider.scss

```diff
@@ -21,30 +22,29 @@ $total_time: $time_per_slide * $num_of_slides; //total time needed for all image
         opacity:1;
         top: 0;
         left: 0;
-
-        &:hover {
-            animation-play-state: paused;
-
-        }
     }
 
-    .slider__img {
+    &__img {
         display: inline-block;
         height: 100%;
         flex: 1;
         background-size: cover;
         background-position: center center;
     }
 
-    .slider__text {
+    &__text {
         padding: 1em;
         display: inline-block;
         flex: 2;
-        p, h3, h4, h5 {
-            text-align: left;
-            color: #3c3c3c;
+    }
 
-        }
+    &__paragraph {
+        text-align: left;
```

Comment:
It means take the class name from above and append here at the front so the actual css class name is `.slider__paragraph` it is shorthand way to wrap when you are within a css class rule